src/schug/load/biomart.py

```diff
@@ -40,14 +40,14 @@ def __init__(self):
         }
 
     @staticmethod
-    def create_biomart_xml(filters: dict, attributes: List[str]) -> str:
+    def create_biomart_xml(filters: dict, attributes: List[str], header: bool) -> str:
         """Convert biomart query params into a xml format biomart query"""
         filter_lines: List[str] = EnsemblXML.xml_filters(filters)
         attribute_lines = EnsemblXML.xml_attributes(attributes)
         xml_lines = [
```

Comment:
I would have agreed if the [the biomart python](https://github.com/sebriois/biomart) didn't exist. I think we should instead switch to that one as soon as we have some time, because it handles all the queries in a much fanciers and easier way (no had coded xml strings for instance). It would be a waste of time to modify our ugly (but working)code to allow more general queries I think.

Reply:
Sounds good